package.json

```diff
@@ -66,25 +66,25 @@
   },
   "devDependencies": {
     "@inrupt/eslint-config-lib": "^2.1.3",
-    "@playwright/test": "^1.31.2",
+    "@playwright/test": "^1.37.1",
     "@rushstack/eslint-patch": "^1.2.0",
     "@types/dotenv-flow": "^3.2.0",
-    "@types/jest": "^28.1.3",
-    "@types/node": "^20.1.3",
-    "@types/node-fetch": "^2.6.2",
+    "@types/jest": "^29.5.4",
+    "@types/node": "^20.5.3",
+    "@types/node-fetch": "^2.6.4",
     "dotenv-flow": "^3.2.0",
-    "jest": "^28.1.1",
-    "jest-environment-node": "^29.4.3",
-    "prettier": "3.0.2",
-    "rimraf": "^5.0.0",
-    "rollup": "^3.15.0",
+    "jest": "^29.6.3",
+    "jest-environment-node": "^29.6.3",
+    "prettier": "^3.0.2",
+    "rimraf": "^5.0.1",
+    "rollup": "^3.28.1",
     "rollup-plugin-typescript2": "^0.35.0",
-    "ts-jest": "^28.0.5",
-    "typescript": "^5.0.4"
+    "ts-jest": "^29.1.1",
+    "typescript": "^5.1.6"
   },
   "dependencies": {
     "node-fetch": "^2.6.7",
-    "undici": "^5.19.1"
+    "undici": "^5.23.0"
```

Comment:
I'm fine with bumping the devDependencies, but I don't want to bump dependencies as we will force consumers to upgrade their undici version if they can't. In most (all?) cases their transitive should be resolving them to the latest minor version anyway.

Reply:
Hopefully a minor change isn't breaking for downstream consumers, but since there aren't any vulnerabilities coming from this version, I don't mind holding it off.